src/template.html

```diff
@@ -6,5 +6,26 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>%= htmlWebpackPlugin.options.title %</title>
   </head>
-  <body></body>
+  <body>
+    <header><h1>Leaderboard</h1></header>
+    <main>
+      <section>
+        <div class="Side">
+          <div class="fstSect">
+            <h3>Recent Scores</h3>
+            <button id="refresh" class="btn">Refresh</button>
+          </div>
+          <ul class="recentScores"></ul>
+        </div>
+        <form autocomplete="off">
+          <label>Add your score</label>
+          <input type="text" id="name" required maxlength="15" /><input type="number" id="score" required maxlength="2" />
```

Comment:
- [ ] I kindly suggest you add placeholders for your inputs so users are aware where to put the score and name 👍